marketdata/stream/conn_nhoory.go

```diff
@@ -19,13 +21,14 @@ type nhooyrWebsocketConn struct {
 func newNhooyrWebsocketConn(ctx context.Context, u url.URL) (conn, error) {
 	ctxWithTimeout, cancel := context.WithTimeout(ctx, 3*time.Second)
 	defer cancel()
-
 	conn, _, err := websocket.Dial(ctxWithTimeout, u.String(), &websocket.DialOptions{
 		CompressionMode: websocket.CompressionContextTakeover,
 		HTTPHeader: http.Header{
 			"Content-Type": []string{"application/msgpack"},
+			"User-Agent":   []string{alpaca.GetVersion()},
```

Comment:
minor: While manipulating directly the underlying map is ok now, using the Set or Add methods of `http.Header` has the benefit of canonicalizing the header key.
> Get assumes that all keys are stored in canonical form. To use non-canonical keys, access the map directly.

Reply:
Addressed: https://github.com/alpacahq/alpaca-trade-api-go/pull/254/commits/858800b73d759b9614842be2439ff1c9e00a3407

Is this what we want?